server/src/test/java/com/google/speech/tools/voxetta/BlobstoreLinkServletTest.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
```

Comment:
filepath for tests should be `javatests/com/google/speech/tools/voxetta/servlets/myservlettest.java`

Tests should be put in a mirror `javatests` directory with the same files and folder structure as the `java` directory.

Reply:
The walkthroughs left out the additional servlets folder and I was never completely sure why--thought it was convention but I think they just forgot it--fixed!